src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -98,19 +108,62 @@ private String getHIUId(ConsentDetail consentDetail) {
 
     public Mono<ConsentApprovalResponse> approveConsent(String authorization,
                                                         String requestId,
-                                                        List<GrantedConsent> grantedConsents) {
+                                                        List<GrantedConsent> grantedConsents,
+                                                        String callBackUrl) {
         String patientId = TokenUtils.readUserId(authorization);
```

Comment:
PatientID should have been sent from controller, follow the same thing for other methods.